modules/settings/commands.py

```diff
@@ -10,27 +10,27 @@ class SettingsCog(discord.Cog):
     def __init__(self, bot):
         self.bot = bot
 
-    settings = discord.SlashCommandGroup("settings", "Setting commands")
+    settings = discord.SlashCommandGroup("settings", "Setting commands", default_member_permissions=discord.Permissions(manage_guild=True))
     level = settings.create_subgroup("level", "Leveling settings")
     logs = settings.create_subgroup("logs", "Log settings")
 
     @level.command()
-    @ext_commands.has_guild_permissions(manage_guild=True)
+    # @ext_commands.has_guild_permissions(manage_guild=True)
     @discord.option("enabled", bool, description="Enable or disable leveling")
     async def toggle(self, ctx: discord.ApplicationContext, enabled):
         await settings_manager.set(scope_type=SettingsManager.SCOPES_GUILD, scope_id=ctx.guild_id, setting_key=SettingKey.LEVEL_UP_ENABLED, value=enabled)
         await ctx.respond(f"Leveling has been {'enabled' if enabled else 'disabled'}!")
 
     @level.command()
-    @ext_commands.has_guild_permissions(manage_guild=True)
+    # @ext_commands.has_guild_permissions(manage_guild=True)
     @discord.option("message", str, description="The new level-up message")
     async def message(self, ctx: discord.ApplicationContext, message):
         await settings_manager.set(scope_type=SettingsManager.SCOPES_GUILD, scope_id=ctx.guild_id, setting_key=SettingKey.LEVEL_UP_MESSAGE, value=message)
         parsed_message = await process_message_with_params(message, user=ctx.author, guild=ctx.guild)
         await ctx.respond(f"Level-up message updated to:\n\n{message}\n\nExample: {parsed_message}")
 
     @logs.command()
-    @ext_commands.has_guild_permissions(manage_guild=True)
+    # @ext_commands.has_guild_permissions(manage_guild=True)
```

Comment:
You should remove those commented-out lines of code as they aren't required and probably won't be